actions/get_log.py

```diff
@@ -0,0 +1,69 @@
+from napalm import get_network_driver
+
+from lib.action import NapalmBaseAction
+
+
+class NapalmGetLog(NapalmBaseAction):
+    """Get Logs from a network device via NAPALM
+    """
+
+    def run(self, hostname, host_ip, driver, port, credentials, lastlines, htmlout=False):
+
+        try:
+            # Look up the driver  and if it's not given from the configuration file
+            # Also overides the hostname since we might have a partial host i.e. from
+            # syslog such as host1 instead of host1.example.com
+            #
+            (hostname,
+             host_ip,
+             driver,
+             credentials) = self.find_device_from_config(hostname, host_ip, driver, credentials)
+
+            login = self.get_credentials(credentials)
+
+            if not lastlines:
+                lastlines = 5
+
+            if driver == 'junos':
```

Comment:
This works - generally I prefer to store this kind of thing in a dict and do a lookup. Like this:

```
{
    'iosxr': {
        'log_cmd': 'show log',
        'commands': ['term width 0', 'term len 0']
    }
}
```